src/Sparrow/Platform/Win32/Win32ElectricFencedMemory.cs

```diff
@@ -31,7 +31,7 @@ internal unsafe class Win32ElectricFencedMemory
 
             var firstWritablePage = virtualAlloc + 4096;
 
-            Win32UnmanagedMemory.Set(firstWritablePage, 0xED, 4096 * sizeInPages); // don't assume zero'ed mem
```

Comment:
Can we remove `Win32UnmanagedMemory.cs` file now?

Reply:
ok, no need to implement them on our own, we're better of by using those syscals I think

Reply:
in 4.2 we replace memove with Buffer.MemoryCopy